### PR TITLE
Update KR Translation for 1.4.0

### PR DESCRIPTION
**Pull request by potatogim**
This PR updates Korean translation for 1.4.0

Signed-off-by: Ji-Hyeon Gim <potatogim@gluesys.com>

### CODE REVIEW

File: assets/i18n/active.ko.json [hunk 1]
```diff
@@ -167,12 +195,24 @@
...
+    "hash": "sha1-213785e569b994863b96b984772e9497f8629bbc",
+    "other": "투표 참여가 집계되었습니다. {{.Remains}}개의 응답이 남았습니다."
+  },
   "response.vote.updated": {
     "hash": "sha1-e23eeb209a3a322e1c1174d7816d2ccbca52080b",
     "other": "투표 참여가 갱신되었습니다."
   }
-}
\ No newline at end of file
+}
```
Comment (hanzei) [inline assets/i18n/active.ko.json]: Please remove the newline

File: assets/i18n/active.ko.json [hunk 2]
```diff
@@ -167,12 +195,24 @@
...
+    "hash": "sha1-fd557010c26787d3aeece2745511dab764cb09c7",
+    "other": "모든 투표가 초기화됩니다. 이전 응답은 [{{.ClearedVotes}}] 였습니다."
+  },
   "response.vote.counted": {
     "hash": "sha1-3420b74a2fee31243e4e4e7a72a527832685f97d",
     "other": "투표 참여가 집계되었습니다."
   },
+  "response.vote.multi.updated": {
+    "hash": "sha1-213785e569b994863b96b984772e9497f8629bbc",
+    "other": "투표 참여가 집계되었습니다. {{.Remains}}개의 응답이 남았습니다."
```
Comment (hanzei) [inline assets/i18n/active.ko.json:212]: Just to double check: The translation is the same for one remaining vote and for multiple ones?

**Review by hanzei — COMMENTED** [review]
Awesome :+1:

File: assets/i18n/active.ko.json (hunk 2 above)
Comment (potatogim) [reply assets/i18n/active.ko.json:212]: Yes :smiley:

**Review by hanzei — APPROVED** [review]

**Pull request merged**